src/Microsoft.DotNet.Tools.Pack/NuGet/PathResolver.cs

```diff
@@ -0,0 +1,301 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace NuGet
+{
+    public static class PathResolver
```

Comment:
I'm not sure we want to keep this code in the long run. We're using a new globbing library that came from ASP.NET 5 for other scenarios but for compat I guess it's fine.